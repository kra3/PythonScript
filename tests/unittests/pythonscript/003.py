# minimal python class
class Parent:

    def __init__(self):
        self.message = 'h√©llo world'

    def say(self):
        print self.message



class Child(Parent):

    def __init__(self):
        Parent.__init__(self)
        self.more = 'how are you?'

    def greeting(self):
        print self.more


c = Child()
c.say()
c.greeting()
