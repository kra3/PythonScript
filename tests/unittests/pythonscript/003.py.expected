h√©llo world
how are you?
